pkg/networkservice/common/timeout/server_test.go

```diff
@@ -141,7 +143,7 @@ func stressTestRequest() *networkservice.NetworkServiceRequest {
 	}
 }
 
-func TestTimeoutServer_StressTest_DoubleClose(t *testing.T) {
+func TestTimeoutServer_StressTest(t *testing.T) {
```

Comment:
How long is this test running? Note: we should expect 1s for the unit test.

Reply:
20ms locally.

Reply:
**build and test (ubuntu-latest)**
```
ok  	github.com/networkservicemesh/sdk/pkg/networkservice/common/serialize	0.896s
```
**build and test (macos-latest)**
```
ok  	github.com/networkservicemesh/sdk/pkg/networkservice/common/serialize	0.760s
```
**build and test (windows-latest)** <-- it is actually very slow for all tests
```
ok  	github.com/networkservicemesh/sdk/pkg/networkservice/common/serialize	1.813s
```